packages/dds/sequence/src/test/sharedIntervalCollection.spec.ts

```diff
@@ -105,10 +97,6 @@ describe("SharedIntervalCollection", () => {
 			collection2.add(10, 30, IntervalType.Simple);
 			assertIntervals(collection1, [{ start: 0, end: 20 }]);
 			assertIntervals(collection2, [{ start: 10, end: 30 }]);
-			assert.equal(Array.from(collection1.findOverlappingIntervals(1, 3)).length, 1);
```

Comment:
seems like it would be nice to not remove this test coverage.

Do the assertions pass if you pass `undefined` for `this.client` and `intervalHelpers` from `packages\dds\sequence\src\intervals\interval.ts` to create an `OverlappingIntervalIndex`?

Reply:
the undefined won't typecheck (you'll need to cast) but that's ok.

Reply:
I re-added the test coverage, using the `overlappingIntervalIndex` with undefined Client and plain-number intervalHelpers